src/core/state/covid-tests/types.ts

```diff
@@ -0,0 +1,27 @@
+type TTestLoggedTodayBase = {
+  currentPatientLoggedToday: boolean;
+};
+
+type TTestLoggedTodayConditional =
+  | {
+      dateLastSeenLFTPrompt?: Date;
+      timesSeenLFTPrompt: 0;
+    }
+  | {
+      dateLastSeenLFTPrompt: Date;
+      timesSeenLFTPrompt: Exclude<number, 0>;
+    };
+
+type TTestLoggedToday = TTestLoggedTodayBase & TTestLoggedTodayConditional;
+
+export type TStateCovidTests = { lftPrompt: { [patientId: string]: TTestLoggedToday } };
+
+export type TUpdateDateLFTPromptAction = {
+  patientId: string;
+  dateLastSeenLFTPrompt: Date;
+};
+
+export type TUpdateCurrentPatientLoggedToday = {
```

Comment:
```suggestion
export type TUpdateCurrentPatientLoggedTodayAction = {
```

to stick to the convention for Actions?